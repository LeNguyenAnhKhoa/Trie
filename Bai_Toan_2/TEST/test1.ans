13
11
13
