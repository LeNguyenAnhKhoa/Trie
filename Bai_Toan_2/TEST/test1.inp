7 3
4 9 1 2 8 2 7
4
2
5
