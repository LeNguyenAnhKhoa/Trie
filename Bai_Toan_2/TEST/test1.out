27
31
31
30
27
30
31
